def nomeErrado(n):
    return 0
